frontend/src/app/projects/models/service/model.service.ts

```diff
@@ -112,6 +112,27 @@ export class ModelService {
       );
   }
 
+  updateModels(
+    projectSlug: string,
+    modelUpdates: { modelSlug: string; patchModel: PatchModel }[],
+  ): Observable<Model[]> {
+    const updateObservables = modelUpdates.map(({ modelSlug, patchModel }) =>
+      this.http.patch<Model>(
+        `${this.base_url}${projectSlug}/models/${modelSlug}/`,
+        patchModel,
+      ),
```

Comment:
I agree that both variants are not ideal right now. One idea to improve this would be to name the function `applyModelPatch` which better captures what is actually done (less generic than `patchModel`).
However its also totally fine to keep it as it is right now.

Reply:
Much nicer, updated